packages/material-ui/src/Box/styleFunction.js

```diff
@@ -72,6 +72,41 @@ const getThemeValue = (prop, value, theme) => {
   return { [prop]: value };
 };
 
+function createEmptyBreakpointObject(breakpoints) {
+  const breakpointsInOrder = breakpoints.keys.reduce((acc, key) => {
+    const breakpointStyleKey = breakpoints.up(key);
+    return {
+      ...acc,
+      [breakpointStyleKey]: {},
+    };
```

Comment:
This would be even faster, we have seen a huge difference when rendering 100,000 rows in https://github.com/dtassone/material-ui-x/pull/3/commits/7a2e78ae7250b320a2ad6a43c77aca3c53c79b73.

```suggestion
    acc[breakpointStyleKey]: {};
    return acc;
```